server/models/Profile.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+  },
+  lastName: {
+    type: String,
+  },
+  description: {
+    type: String,
+  },
+  availability: {
```

Comment:
@rajivtitus  As it is now the availability property is defined as an array of Dates. Would this not suffice for weekly availability since it is [straightforward to convert a JS date object to a day of the week](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) and [would be relatively easy to query](https://stackoverflow.com/a/18149149/15179819) when a user looks for sitters on a specific date(as per the Dog Owner user journey).

Reply:
@hodge-jai This design will work but we won't be able to specify start/end times within each date.